src/pages/Pokemon/index.js

```diff
@@ -26,6 +27,19 @@ const Pokemon = () => {
     const catSpec = await cat.json();
     item.category = catSpec?.genera[7]?.genus;
 
+    const typeInfo1 = await fetch(item.types[0].type.url);
+    const typeStuff1 = await typeInfo1.json();
```

Comment:
we could rename these to `typeInfo` and `typeInfoJSON` or something to make it less confusing. Same for the other two below mate